docs/example/README.md

```diff
@@ -28,65 +42,142 @@ Use the [installation guide](/docs/installation_guide.md) to install the `osm` c
 osm install
 ```
 
-## Install Applications
+## Deploying the Bookstore Demo Applications
 The `Bookstore`, `Bookbuyer`, `Bookthief`, `Bookwarehouse` demo applications will be installed in their respective Kubernetes Namespaces. In order for these applications to be injected with a Envoy sidecar automatically, we must add the Namespaces to be monitored by the mesh.
 
+### Create the Bookstore Application Namespaces
 ```bash
-kubectl create ns bookstore
-kubectl create ns bookbuyer
-kubectl create ns bookthief
-kubectl create ns bookwarehouse
-osm namespace add bookstore bookbuyer bookthief bookwarehouse
+for i in bookstore bookbuyer bookthief bookwarehouse; do kubectl create ns $i; done
```

Comment:
Please revert to the previous, this looks harder to read and is fine to have in the automated script but not as instructions. We don't want users to be understanding how bash for loop syntax should be interpreted.

Reply:
I want to make the experience as quick and easy as possible. This is a simple bash for loop. I would expect someone looking to run this to be comfortable with this.

Reply:
Its definitely simple for people familiar with bash, but should we assume that in the manual demo guide?